### PR TITLE
[FrameworkBundle][HtmlSanitizer] Fix calling `allowStaticElements` when setting `allow_all_static_elements: true`

### PR DESCRIPTION
**Pull request by norkunas**
| Q             | A
| ------------- | ---
| Branch?       | 6.1
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | [symfony/symfony-docs#... ](https://github.com/symfony/symfony-docs/pull/16838)

```
Symfony\Component\ErrorHandler\Error\UndefinedMethodError {#21964
  #message: """
    Attempted to call an undefined method named "allowAllStaticElements" of class "Symfony\Component\HtmlSanitizer\HtmlSanitizerConfig".\n
    Did you mean to call e.g. "allowSafeElements" or "allowStaticElements"?
    """
  #code: 0
  #file: "./var/cache/dev/ContainerXJ5lfcV/getHtmlSanitizer_Sanitizer_PostContent_SanitizerService.php"
  #line: 25
  trace: {
    ./var/cache/dev/ContainerXJ5lfcV/getHtmlSanitizer_Sanitizer_PostContent_SanitizerService.php:25 {
      ContainerXJ5lfcV\getHtmlSanitizer_Sanitizer_PostContent_SanitizerService::do($container, $lazyLoad = true)
      › $a = new \Symfony\Component\HtmlSanitizer\HtmlSanitizerConfig();
      › $a = $a->allowAllStaticElements();
      › $a = $a->forceHttpsUrls(false);
    }
    ./var/cache/dev/ContainerXJ5lfcV/App_KernelDevDebugContainer.php:981 { …}
    ./var/cache/dev/ContainerXJ5lfcV/getTestCommandService.php:23 { …}
    ./var/cache/dev/ContainerXJ5lfcV/App_KernelDevDebugContainer.php:981 { …}
    ./vendor/symfony/dependency-injection/Container.php:383 { …}
    ./vendor/symfony/dependency-injection/Argument/ServiceLocator.php:43 { …}
    ./vendor/symfony/console/CommandLoader/ContainerCommandLoader.php:46 { …}
    ./vendor/symfony/console/Application.php:570 { …}
    ./vendor/symfony/console/Application.php:657 { …}
    ./vendor/symfony/framework-bundle/Console/Application.php:114 { …}
    ./vendor/symfony/console/Application.php:259 { …}
    ./vendor/symfony/framework-bundle/Console/Application.php:80 { …}
    ./vendor/symfony/console/Application.php:171 { …}
    ./vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php:54 { …}
    ./vendor/autoload_runtime.php:29 { …}
    ./bin/console:15 { …}
  }
}
```

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -2683,7 +2683,7 @@ private function registerHtmlSanitizerConfiguration(array $config, ContainerBuil
             }
 
             if ($sanitizerConfig['allow_all_static_elements']) {
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: what about renaming this to allow_static_elements then? we can still do it imho

**Review by tgalopin — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php (hunk 1 above)
Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: 👍 for me

Comment (norkunas) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: Updated

**Review by OskarStark — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
(please check the ci failures)

**Comment by norkunas** [comment]
> (please check the ci failures)

Not sure how to fix these errors because I've already updated all places.

**Comment by stof** [comment]
Well, according to the CI failure, you haven't updated the XML test for that config

**Comment by norkunas** [comment]
> Well, according to the CI failure, you haven't updated the XML test for that config

Found it, thanks :)

**Review by stof — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by chalasr** [comment]
Thank you @norkunas.

**Pull request merged**